backend/v2/epochs/dependencies.py

```diff
@@ -61,7 +56,48 @@ def get_epochs_subgraph(
     return EpochsSubgraph(settings.subgraph_endpoint)
 
 
+async def get_current_epoch(epochs_contracts: GetEpochsContracts) -> int:
+    return await epochs_contracts.get_current_epoch()
+
+
+async def get_indexed_epoch(epochs_subgraph: GetEpochsSubgraph) -> int:
+    sg_epochs = await epochs_subgraph.get_epochs()
+    sg_epochs_sorted = sorted(sg_epochs, key=lambda d: d.epoch_num)
+
+    return sg_epochs_sorted[-1].epoch_num
```

Comment:
This looks like could be done by single query to subgraph, right? Order by + limit on the graphql query

Reply:
kk